src/install/index.js

```diff
@@ -19,33 +18,60 @@ const installer = utils.getInstallerPath()
  */
 function install () {
   return new Promise((resolve, reject) => {
-    const tailer = new Tailer()
+    const buid_tools_tailer = new Tailer(utils.getBuitToolsInstallerPath().logPath)
+    const python_tailer = new Tailer(utils.getPythonInstallerPath().logPath, 'ucs2') // The log file for msiexe is utf-16
 
-    tailer.on('exit', (result, details) => {
+
+    buid_tools_tailer.on('exit', (result, details) => {
+      debug("build tools tailer exited");
       if (result === 'error') {
         debug('Installer: Tailer found error with installer', details)
         reject(err)
       }
 
       if (result === 'success') {
         console.log(chalk.bold.green('Successfully installed Visual Studio Build Tools.'))
-        debug('Installer: Successfully installed according to tailer')
+        debug('Installer: Successfully installed Visual Studio Build Tools according to tailer')
         resolve()
       }
 
       if (result === 'failure') {
         console.log(chalk.bold.red('Could not install Visual Studio Build Tools.'))
         console.log('Please find more details in the log files, which can be found at')
-        console.log(path.join(installer.directory))
+        console.log(utils.getWorkDirectory())
         debug('Installer: Failed to install according to tailer')
         resolve()
       }
     })
 
+    python_tailer.on('exit', (result, details) => {
+      debug("python tailer exited");
+      if (result === 'error') {
+        debug('Installer: Tailer found error with installer', details)
+        reject(err)
+      }
+
+      if (result === 'success') {
+        console.log(chalk.bold.green('Successfully installed Python 2.7'))
+        debug('Installer: Successfully installed Python 2.7 according to tailer')
+        // TODO: Actually get the path from the log file. This is for the case that python 2.7 is already installed 
+        resolve({pythonPath: utils.getPythonInstallerPath().targetPath})
```

Comment:
Need to find a better way to pass the environmental variables to the setEnvironment() function